### PR TITLE
util: add metrics for channel full.

### PR DESCRIPTION
**Pull request by disksing**
For #1146

Because we found no way to get a channel's size, only monitor the channel full error.

@siddontang @BusyJay

### CODE REVIEW

File: src/server/node.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ impl<C> Node<C>
             store.set_address(cfg.advertise_addr.clone())
         }
 
-        let ch = SendCh::new(event_loop.channel());
+        let ch = SendCh::new(event_loop.channel(), "node");
```
Comment (siddontang) [inline src/server/node.rs]: the channel name is raftstore here.

File: src/server/server.rs [hunk 2]
```diff
@@ -110,7 +110,7 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
                                  EventSet::readable(),
                                  PollOpt::edge()));
 
-        let sendch = SendCh::new(event_loop.channel());
+        let sendch = SendCh::new(event_loop.channel(), "server");
```
Comment (siddontang) [inline src/server/server.rs]: the channel name is raft-server here

**Comment by siddontang** [comment]
Rest LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**